pi_sht1x/logging.py

```diff
@@ -1,9 +1,10 @@
-from os import path
+import os
```

Comment:
I'd prefer to use the previous import to scope the import to path, which is all we need here.